README.md

```diff
@@ -39,6 +39,7 @@ The Pravega Operator manages Pravega clusters deployed to Kubernetes and automat
 - Kubernetes 1.9+
 - Helm 2.10+
 - An existing Apache Zookeeper 3.5 cluster. This can be easily deployed using our [Zookeeper operator](https://github.com/pravega/zookeeper-operator)
+- An existing Apache Bookkeeper 4.7.3 cluster. This can be easily deployed using our [BookKeeper Operator](https://github.com/pravega/bookkeeper-operator)
```

Comment:
This version should be updated, right? We now moved to Bookkeeper 4.9.2.

Reply:
fixed